packages/driver/cypress/integration/commands/navigation_spec.js

```diff
@@ -2177,11 +2177,14 @@ describe('src/cy/commands/navigation', () => {
               > ${error}\n
             Before the page load, you were bound to the origin policy:\n
               > http://localhost:3500\n
+            In order to navigate to a different origin, you can enable the \`experimentalSessionAndOrigin\` flag and use \`cy.origin()\`:\n
+            \`cy.get('.goToAnotherOrigin').click()\`
+            \`cy.origin('http://localhost:3500', () => {\`
+            \`  <commands targeting http://localhost:3500 go here>\`
```

Comment:
I'm not sure I've seen bracket syntax in our error messages. have we used JS comment syntax at all?

Reply:
This is the syntax we've used for the other cross origin error messages. I don't see comment syntax being used in the other error messages. The only other message I see uses `...`:
```
describe('{{title}}', { retries: {{numRetries}} }, () => {
  ...
})
```